Sources/Fluent/Relations/RelationError.swift

```diff
@@ -3,6 +3,7 @@
 /// Ex: Children, Parent, Siblings
 public enum RelationError {
     case idRequired(Entity)
+    case oneToOneConstraint(Entity, Entity.Type, Int)
```

Comment:
Adding one-to-one support to fluent will take a bit more work than just adding an error if the count is more than 1. This should be in a separate PR.